CONTRIBUTING.md

```diff
@@ -33,6 +33,11 @@ While we will try to be as helpful as we can on any issue reported, please inclu
 
 Creating a good reproduction really helps contributors investigate and resolve your issue quickly. In many cases, the act of creating a minimal reproduction illuminates that the source of the bug was somewhere outside the library in question, saving time and effort for everyone.
 
+### Current branches
+
+1. **Federation 2 (alpha)** is the current `main` branch.
+2. **Prior releases** are located under the `version-0.x` branch.  This comprises all 0.x packages previously released.  Installing `latest` from npm will pull from here.
```

Comment:
I might suggest putting this on the `README` too! (It complements or might supersede what's there right now!)